rendezvous/README.md

```diff
@@ -0,0 +1,105 @@
+# Rendezvous Service
+
+Scope:
+- real-time applications that require rendezvous
+- replace ws-star-rendezvous with a rendezvous service daemon and a fleet
+  of p2p-circuit relays.
+
+## Rendezvous Protocol
+
+The rendezvous protocol provides facilities for real-time peer discovery within
+application specific namespaces. Peers connect to the rendezvous service and register
+their presence in one or more namespaces. Registrations persist until the peer
+disconnects or explicitly unregisters.
+
+Peers can enter rendezvous and dynamically receive announcements about peer
+registrations and unregistrations within their namespaces of interest.
+For purposes of discovery (eg bootstrap), peers can also ask the service for
+a oneof list of peers within a namespace.
+
+### Interaction
+
+Client peer `A` connects to the rendezvous service `R` and registers for namespace
+`my-app` with a `REGISTER` message. It subsequently enters rendezvous with
+a `RENDEZVOUS` message and waits for `REGISTER`/`UNREGISTER` announcements from
```

Comment:
I'm not sold on having an explicit Rendezvous thingy. Isnt it technically just the same as polling the Discovery protocol?

Reply:
it's a more efficient version of that, as it listens for new peers on top of the initial lookup:
- `DISCOVER` returns a list of current peers and stops 
- `RENDEZVOUS` returns the current peers and then sends deltas (new `REGISTER/UNREGISTER`).

Reply:
So it's pull -vs- push.